Code/GraphMol/FMCS/MaximumCommonSubgraph.cpp

```diff
@@ -600,8 +600,8 @@ struct AtomMatch {  // for each seed atom (matched)
 };
 typedef std::vector<AtomMatch> AtomMatchSet;
 
-std::string MaximumCommonSubgraph::generateResultSMARTS(
-    const MCS& mcsIdx) const {
+std::string MaximumCommonSubgraph::generateResultSMARTSAndQueryMol(
```

Comment:
The pointer to a pointer is kind of gross. How about either taking the `ROMOL_SPTR` directly or, preferably, returning an `std::pair`?

Reply:
Yeah, no point, especially given it's a private function.